apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/MapAssertionSamples.kt

```diff
@@ -0,0 +1,249 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class MapAssertionSamples {
```

Comment:
please rename to MapExpectationSamples and move to package ch.tutteli.atrium.api.infix.en_GB.samples